electrum/gui/qt/qrcodewidget.py

```diff
@@ -20,7 +20,8 @@ def __init__(self, data = None, fixedSize=False):
         QWidget.__init__(self)
         self.data = None
         self.qr = None
-        self.margin = 0
+        
+        self.margin = 1
```

Comment:
> Set `QRCodeWidget` margin not 0 since in dark mode the
> default created QR code is not properly scanned.

Hmm, good observation. Still, it's an independent issue. Could you please split this out into a separate PR?

Reply:
(done in https://github.com/spesmilo/electrum/pull/7813)